template.jinja/confreg/session_feedback.html

```diff
@@ -50,8 +50,15 @@ <h2>Slides</h2>
 <input type="submit" name="submit" value="Upload slides">
 </form>
 
-{%if feedbackcount%}
 <h2>Feedback</h2>
+<h3>Feedback link</h3>
```

Comment:
I wasn't sure if this should be conditional on if speaker feedback is open.  We want the QR code to be available early enough so that speakers include it in their slide deck, so long before the conference starts

Reply:
I think it's reasonable to show the QR code as soon as the session is approved (but not for sessions that are not approved, such as pending or rejected of course). 

But but instead we should remove the QR code maybe on the combination of "speaker feedback is closed but we are past the end date of the conference"?

Reply:
Sorry to not catch that in my first high level overview, but -- comparisons should be made on the status code, not the status string. In case we ever (unlikely, I know) get a translation. But also, that's the pattern that other comparison use.

I think we should also try to avoid *generating* the qrcode if it's not going to be used, as that's probably the more expensive of all the parts of generating the page. So maybe putting the filter in the code to only generate it when it's needed, and then conditional it on whether it was generated or not, could be a way forward from that that would solve both those at once?